StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsOAuthPrepane.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FinancialConnectionsOAuthPrepane.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 1/6/23.
+//
+
+import Foundation
+@_spi(STP) import StripeUICore
+
+struct FinancialConnectionsOAuthPrepane: Decodable {
+
+    let institutionIcon: FinancialConnectionsImage?
+    let title: String
+    let body: OauthPrepaneBody
+    let partnerNotice: OauthPrepanePartnerNotice?
+    let cta: OauthPrepaneCTA
+    let dataAccessNotice: FinancialConnectionsDataAccessNotice
+
+    struct OauthPrepaneBody: Decodable {
+        let entries: [OauthPrepaneBodyEntry]?
```

Comment:
backend for double check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/oauth_prepane_body.rb#L7